src/main/kotlin/org/wfanet/measurement/kingdom/deploy/gcloud/spanner/testing/KingdomDatabasesProviderRule.kt

```diff
@@ -21,25 +21,15 @@ import org.wfanet.measurement.common.identity.RandomIdGenerator
 import org.wfanet.measurement.common.testing.ProviderRule
 import org.wfanet.measurement.gcloud.spanner.AsyncDatabaseClient
 import org.wfanet.measurement.gcloud.spanner.testing.SpannerEmulatorDatabaseRule
-import org.wfanet.measurement.integration.common.InProcessKingdom
-import org.wfanet.measurement.kingdom.deploy.gcloud.spanner.SpannerKingdomRelationalDatabase
-import org.wfanet.measurement.kingdom.deploy.gcloud.spanner.testing.KINGDOM_SCHEMA
-import org.wfanet.measurement.kingdom.deploy.gcloud.spanner.testing.SpannerDatabaseTestHelper
+import org.wfanet.measurement.kingdom.db.testing.KingdomDatabases
 
-class KingdomDatabasesProviderRule : ProviderRule<InProcessKingdom.Databases> {
+class KingdomDatabasesProviderRule : ProviderRule<KingdomDatabases> {
   private val spannerDatabase = SpannerEmulatorDatabaseRule(KINGDOM_SCHEMA)
   private val clock = Clock.systemUTC()
   private val idGenerator = RandomIdGenerator(clock)
```

Comment:
Any chance we are concerned about the very very low chance of collisions here?

Reply:
Nope -- if there is a collision, the insert will fail and it'll be treated as a transient error. But also, I think the probability of collision after issuing 1M ids is around 0.00000271% (approximated using `1 - e ^ (1M^2 / (2^65))`).